Week1/homework/index.js

```diff
@@ -0,0 +1,205 @@
+'use strict';
+
+const mysql = require('mysql');
+const express = require('express');
+const bodyParser = require('body-parser');
+const { promisify } = require('util');
+
+const app = express();
+app.use(bodyParser.json());
+
+const data = require('./cities.json');
+
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'root',
+  password: 'password',
+  database: 'world',
+});
+
+connection.connect();
+
+const createCountries = `CREATE TABLE countries (
+  Name varchar(254) ,
+  Continent varchar(50),
+  Region varchar(50) ,
+  SurfaceArea float(2) ,
+  IndepYear varchar(50) ,
+  Population float(2)  ,
+  LifeExpectancy varchar(50) ,
+  GNP float(2) ,
+  GNPOld varchar(50) ,
```

Comment:
Why gnp float and not gnpOld ?

Reply:
Mysql gave me an error after that I changed it. 👍  I will try again with float.

Reply:
I think you should not use '' around the value if it's float or integer because it is used for string (varchar). Try with removing the quote when inserting into the db to see if you are still getting error.

Reply:
Yes, you are right. I can use float(2). It is okay now. I will update it again. :)